hooks/990-ensure-uids-gids.chroot

```diff
@@ -28,10 +28,6 @@ echo "Ensure no user 107 (ex-docker)"
 cut -d: -f3 /etc/passwd | grep -q 107
 rc=$?; [ "$rc" = "0" ] && MISMATCH=1
 
-echo "Ensure no group 113 (ex-docker)"
```

Comment:
Why this has been removed ?

Reply:
There is a new group added in impish nowadays it seems, the `sgx` group - and it seems to use the 113 GID. So it feels to me that just erroring out on this here doesn't make sense, since it's like a completely new series.